src/methods/photos/index.ts

```diff
@@ -0,0 +1,77 @@
+import urlHelpers from 'url';
+import { handleFeedResponse } from '../../helpers/feed';
+import * as Query from '../../helpers/query';
+import { createRequestParams } from '../../helpers/request';
+import { castResponse } from '../../helpers/response';
+import { isDefined } from '../../helpers/typescript';
+import { OrientationParam, PaginationParams } from '../../types/request';
+
+type PhotoId = {
+  photoId: string;
+};
+
+const PHOTOS_PATH_PREFIX = '/photos';
+
+export const list = {
+  handleRequest: createRequestParams((feedParams: PaginationParams = {}) => ({
+    pathname: PHOTOS_PATH_PREFIX,
+    query: Query.getFeedParams(feedParams),
+  })),
+  handleResponse: handleFeedResponse<any>(),
+};
+
+export const get = {
+  handleRequest: createRequestParams(({ photoId }: PhotoId) => ({
```

Comment:
Re. https://github.com/unsplash/unsplash-web/pull/5900#discussion_r522105536

Are we able to re-use these `handleRequest` functions in our own client now?

Reply:
Yeah, that's the plan! I wanted to make sure before merging this PR but the refactoring work in our client is a bit too involved due to the new `createRequestParams` signature etc. I decided to merge this and then we'll tackle the changes in the other repo. If anything is missing here we can always make adjustments (no breaking changes since it's all under-the-hood functionality)